src/applications/mhv-medical-records/containers/Allergies.jsx

```diff
@@ -94,20 +99,31 @@ const Allergies = props => {
     updatePageTitle,
   );
 
+  const RecordsIndicator = generateNewRecordsIndicator(
```

Comment:
Rather than call this `RecordsIndicator`, which makes it look like a component, can you can call it `lastUpdatedText`.